frontEnd/front-end/src/Components/AskQuestionPage.js

```diff
@@ -10,7 +11,7 @@ const AskQuestionPage = () => {
             <Header/>
             <label className="title_font m-4 pl-5">
             <i class="fas fa-question-circle"></i>
```

Comment:
Use `className` instead of `class` in React components. I think this might be happening in a few other components as well. Another one to look out for is using `htmlFor` instead of `for` in your `<label>` tags on forms :)

Reply:
Thank you. All components has been checked as you mention it